modules/ROOT/pages/dw-multipart-functions-field.adoc

```diff
@@ -166,7 +166,7 @@ Multipart::field("order", myOrder,"application/json", "order.json")`
 [%header, cols="1,3"]
 |===
 | Name | Description
-| `opts` a| A set of parameters that specify:
+| `opts` a | A set of parameters that specify:
```

Comment:
TODO: CHECK if space after "a" messes up the table; probably does. Better to have table directive [%header, cols="1,3a"], with the a here instead of in the table itself. (instance # 1)

Reply:
DONE